public/src/client/topic/events.js

```diff
@@ -130,10 +130,12 @@ define('forum/topic/events', [
 		};
 
 		templates.parse('partials/topic/post-editor', editData, function(html) {
-			translator.translate(html, function(translated) {
-				html = $(translated);
-				editorEl.replaceWith(html);
-				html.find('.timeago').timeago();
+			require(['translator'], function(translator) {
+				translator.translate(html, function(translated) {
+					html = $(translated);
+					editorEl.replaceWith(html);
+					html.find('.timeago').timeago();
+				});
```

Comment:
Of course. Also remove *translator* from line 4 comment and add to *define* like this?
```javascript
/* globals config, app, ajaxify, define, socket, templates, utils */

define('forum/topic/events', [
	'forum/topic/postTools',
	'forum/topic/threadTools',
	'forum/topic/posts',
	'components',
	'translator'
], function(postTools, threadTools, posts, components, translator) {
```